DAO/DAO.py

```diff
@@ -1,16 +1,17 @@
 import psycopg2
 import psycopg2.extras
 
-from config.config import BaseConfig
+from config.config import ProductionConfig
 
 
 class DAO:
 
     def __init__(self):
-        config = BaseConfig()
+        config = ProductionConfig()
 
-        self.conn = psycopg2.connect(dbname=config.DBNAME, user=config.USER, password=config.PASSWORD, host=config.HOST,
-                                     port=config.PORT)
+        self.conn = psycopg2.connect(dbname=config.DATABASE['DBNAME'], user=config.DATABASE['USER'],
+                                     password=config.DATABASE['PASSWORD'], host=config.DATABASE['HOST'],
+                                     port=config.DATABASE['PORT'])
 
     def get_cursor(self):
         return self.conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
```

Comment:
```python
return self.conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
```
This way we don't have to loop for results, it will automatically do it for us.